packages/ui/src/components/Loader/texts.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useCallback, useMemo } from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+import Proptypes from 'prop-types';
+
+import { useInterval, utils } from '@monkvision/toolkit';
+
+const { spacing, shadow } = utils.styles;
+
+const styles = StyleSheet.create({
+  text: {
+    textAlign: 'center',
+    padding: spacing(2),
+    marginVertical: spacing(2),
+    marginHorizontal: spacing(1),
+    color: '#A0A3A6',
+  },
+  playgraound: {
+    ...shadow(6, 0.1),
+    backgroundColor: '#FFF',
+    borderRadius: 22,
+  },
+});
+
+const DELAY = 3000;
+/**
+ * @param texts
+ * @return {JSX.Element}
+ * @constructor
+ */
+export default function Texts({ texts }) {
+  const [currentText, setCurrentText] = useState(texts[0]);
+
+  const nextText = useMemo(() => {
+    const currentIndex = texts.indexOf(currentText);
+    return texts[currentIndex + 1];
+  }, [currentText, texts]);
+
+  const delay = nextText ? DELAY : null;
```

Comment:
Maybe merge the memo like that `[nextText, delay] = useMemo(...)`

Reply:
It was like that at the beginning, and it looked weird to me.. for some reason... will bring it back